pep-0646.rst

```diff
@@ -446,8 +455,26 @@ instance is *not* allowed:
 
 ``*args`` is the only case where an argument can be annotated as ``*Ts`` directly;
 other arguments should use ``*Ts`` to parameterise something else, e.g. ``Tuple[*Ts]``.
-If ``*args`` itself is annotated as ``Tuple[*Ts]``, the old behaviour still applies:
-all arguments must be a ``Tuple`` parameterised with the same types.
+
+Using a Tuple of Types
+''''''''''''''''''''''
+
+Similarly, ``*args`` can be annotated as a star-prefixed type tuple:
+
+::
+
+    def foo(*args: *Tuple[int, int]): ...
+    # Very similar to:
+    def bar(arg1: int, arg2: int): ...
+    # But a) `foo` may be more convenient if e.g. passing `args` on
+    # to something else, and b) `foo` disallows keyword arguments.
+
+Note that using an arbitrary-length type tuple, e.g. ``*args: *Tuple[int, ...]``,
+is *not* valid here as it would be exactly the same as ``*args: int``.
```

Comment:
So we're changing this to claiming it *is* valid here. Maybe we could say that it is "roughly" the same as `*args: int` rather than *exactly*? (Even if we can't think of any differences in the type system, at the very least the thing found in `__annotations__['args']` would be different, so technically it's not *exactly* the same.)

Reply:
Done and done.